wire-schema/src/main/java/com/squareup/wire/schema/EnclosingType.java

```diff
@@ -85,4 +86,11 @@
     }
     return new EnclosingType(location, type, documentation, retainedNestedTypes);
   }
+
+  MessageElement toElement() {
```

Comment:
It's not public, and there's about 20 of these methods on every Schema type that also lack documentation...